wgpu-core/src/command/render.rs

```diff
@@ -1262,6 +1275,31 @@ impl<G: GlobalIdentityHandlerFactory> Global<G> {
                         raw.draw_indexed_indirect(&buffer.raw, offset, 1, 0);
                     }
                 }
+                RenderCommand::PushDebugGroup {
+                    color,
+                    len,
+                    phantom_offsets,
+                } => unsafe {
+                    let (new_peeker, label) =
+                        { phantom_offsets.decode_unaligned(peeker, len, raw_data_end) };
+                    peeker = new_peeker;
+
+                    let str_label = std::ffi::CStr::from_bytes_with_nul_unchecked(label);
```

Comment:
Why don't we do `str::from_utf8` instead, and avoid encoding the trailing nul into the stream? It's redundant anyway, since we encode the count of bytes.